module/Calendar.js

```diff
@@ -0,0 +1,321 @@
+/**
+ * @typedef {object} event
+ * @property {string} event.date - date in format '2022-04-06'.
+ * @property {string} event.time - time in format '12:00:00'.
+ * @property {string} event.title - title of event.
+ */
+
+class Calendar {
+  _events = [];
+
+  /**
+   * @description The method generates and returns a uniqId.
+   * @returns {string} unique Id
+   **/
+  _generateUniqId() {
```

Comment:
I think this method does not belong to the calendar class, it can be moved to utilities.